.github/workflows/ci.yaml

```diff
@@ -278,7 +278,7 @@ jobs:
     timeout-minutes: 120
     env:
       COMPAT: 1
-      BITCOIN_VERSION: 24.1
+      BITCOIN_VERSION: 25.0
```

Comment:
these might have to be strings? Looks like it's truncating to "25" https://github.com/ElementsProject/lightning/actions/runs/5089985776/jobs/9149026391?pr=6280#step:5:18

Reply:
Pushed strings.